.github/workflows/cd.yml

```diff
@@ -179,6 +175,56 @@ jobs:
       - name: cleanup templates
         run: rm -rf ./*.zip && git clean -df
 
+      - name: trigger simpleauth azure build pipeline for stable release
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'simpleauth') && github.event_name == 'workflow_dispatch' && github.event.inputs.preid == 'stable' }}
+        uses: Azure/pipelines@v1
+        with:
+          azure-devops-project-url: 'https://dev.azure.com/mseng/VSIoT'
+          azure-pipeline-name: 'Build Simple Auth Release Artifacts' 
+          azure-devops-token: '${{ secrets.ADO_PAT }}'
+      
+      - name: check build status and download assets for stable release
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'simpleauth') && github.event_name == 'workflow_dispatch' && github.event.inputs.preid == 'stable' }}
+        run: |
+          output=$(./.github/scripts/simpleauth-azdo-pipelines.sh ${{ secrets.ADO_PAT }} 11303 >&1)
+          if [ -z $output ]; then exit 1; fi
+          echo $output
+          asset_id=$(echo $output | rev |cut -d ' ' -f 1 | rev)
+          simpleAuthVer=$(echo ${{ steps.version-change.outputs.CHANGED }} | grep simpleauth | cut -d '@' -f2)
+          curl "https://dev.azure.com/mseng/_apis/resources/Containers/$asset_id/drop?itemPath=drop/Microsoft.TeamsFx.SimpleAuth_$simpleAuthVer.zip" -L -O -u :${{ secrets.ADO_PAT }}
+          if [ ! -f Microsoft.TeamsFx.*.zip ]; then exit 1; fi
+      
+      - name: build SimpleAuth on prerelease
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'simpleauth') && github.event.inputs.preid != 'stable' }}
```

Comment:
CD for alpha and rc, simpleauth can build local, and there no need to trigger the AZDO pipeline